### ISSUE TITLE
A11y_AzureDataStudio_Notebooks - Run_Screen Reader: Narrator is not announcing output of the code cell

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18586]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info and Relationship; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2.Launch Azure Data Studio.
3.Open an existing notebook that has code cells in it.
4. Click Run Cells on Notebook toolbar.
5. Verify the output of the code cell is announced by narrator or not

### Actual Result:
Narrator is not announcing output of the code cell

### Expected Result:
Narrator should announce output of the code cell

### Attachment: 

**Narrator Attachment :**

![A11y_AzureDataStudio_Notebooks - Run_Screen Reader_Narrator](https://user-images.githubusercontent.com/100189407/156108331-564311f1-66b6-48ff-9070-648879f5b6c6.gif)

**NVDA Attachment :**

![A11y_AzureDataStudio_Notebooks - Run_Screen Reader_NVDA](https://user-images.githubusercontent.com/100189407/156108390-fde2c7cb-f8c2-4c27-88f7-e54e3121afa1.gif)

**Comment by alanrenmsft** [comment]
@lucyzhang929 do we have an ETA for this?

**Comment by lucyzhang929** [comment]
I haven't had a chance to look at it yet but I will try to get to it this week.

**Comment by lucyzhang929** [comment]
@alanrenmsft when is the deadline for this?

**Comment by alanrenmsft** [comment]
4/28 according to the date the issue was opened. @lucyzhang929

**Comment by VijayaBharathyA** [comment]
Verified in the latest Environment 
azure Data studio Version: 1.37.0 
Issue is Fixed
![18586_Narrator](https://user-images.githubusercontent.com/100189407/178205124-b19574bb-e723-428f-a4a3-86efd72ce080.jpg)
![18586_NVDA](https://user-images.githubusercontent.com/100189407/178205135-19774f46-3893-4e65-bccf-44a2eae1ede6.jpg)
